### ISSUE TITLE
ANDI's Output different from the screen reader's: One answer missing from an FAQ

### ISSUE THREAD

**Issue by spanchang** [w3c/wcag3#337]
Refer to https://www.ssa.gov/accessibility/andi/help/faq.html
Q. Why is ANDI's Output different from the screen reader's?
Comment:
One reason that is mentioned in the tutorial is omitted from the answers listed against this question: 
If the accessible name for button contains the word  "button", a screen reader appears to speak that word twice. In fact the screen reader is doing its job as expected: voice the role. The accessibility name in fact is the culprit which should exclude the role from the accessible name.

**Comment by fstrr** [comment]
Did you mean to post an issue in the [ANDI repo](https://github.com/SSAgov/ANDI/issues)?

**Comment by spanchang** [comment]
Yes, this is ANDI related. Thanks for flagging.
Sailesh Panchang

Email: ***@***.***
Deque Systems Inc | - Accessibility for Good | www.deque.com
